ui/admin/app/templates/scopes/scope/sessions/index.hbs

```diff
@@ -114,7 +114,10 @@
                 </time>
               </row.cell>
               <row.cell>
-                <Rose::Badge>{{sessionAggregate.session.status}}</Rose::Badge>
+                <SessionStatus
+                  @status={{sessionAggregate.session.status}}
+                  @model={{sessionAggregate.session}}
+                />
```

Comment:
What do you think about passing only session, making the component session-aware?  We're already calling it a SessionStatus component and it lives inside `core`, so I think it's ok.

Reply:
yup makes sense!